qatrfm/environment.py

```diff
@@ -102,6 +102,11 @@ def clean(self):
         shutil.rmtree(self.workdir)
         self.logger.success("Environment clean")
 
+    def get_output(self, variable):
+        output = libutils.execute_bash_cmd(
+            "terraform output -json", cwd=self.workdir)
```

Comment:
why not using `"terraform output -json" + variable` ?

then you just need `json.loads(output)['value'][0]`

and avoid asking terraform to print the full output everytime, which might be a bit slower than asking for a single parameter.